test/unit/gateways/cyber_source_test.rb

```diff
@@ -568,6 +568,15 @@ def test_successful_auth_request
     assert response.test?
   end
 
+  def test_successful_reconciliation_id_2
```

Comment:
Out of an abundance of caution to prove that we are not breaking anything, can you also add a test that shows in the case there is only one reconciliationID present, it is still returned, and reconciliationID2 is not added to the response 😅 

I think the `successful_authorization_response` should work for this use case.